Flaky XML Validation

Recently one of our [pipelines](https://github.com/objectionary/jeo-maven-plugin/actions/runs/13932466764/job/38992732084?pr=1043) failed due:

```
[INFO]   The build exited with code 1. See /home/runner/work/jeo-maven-plugin/jeo-maven-plugin/target/its/annotations/build.log for details.
[INFO]           annotations/pom.xml .............................. FAILED (21.61 s)
...
[ERROR] *  annotations/pom.xml
```

Particularly:

```
[Fatal Error] Math.xmir:1:1: Content is not allowed in prolog.
...
20:05:51 [WARN] com.jcabi.xml.StrictXML: 1 XML validation error(s):
  cvc-elt.1.a: Cannot find the declaration of element 'program'.
```

Seems, that it's a problem with XML validation. We should investigate the issue and suggest a solution.

--- a/pom.xml
+++ b/pom.xml
@@ -320,7 +320,6 @@
           <pomExcludes>
             <exclude>spring-fat/pom.xml</exclude>
           </pomExcludes>
-          <parallelThreads>10</parallelThreads>
         </configuration>
       </plugin>
       <plugin>
@@ -527,7 +526,6 @@
                 All integration tests are enabled in the 'long' profile.
               -->
               <pomExcludes/>
-              <parallelThreads>10</parallelThreads>
             </configuration>
           </plugin>
         </plugins>

